.gitignore

```diff
@@ -0,0 +1,18 @@
+# IntelliJ project files
+.idea
+*.iml
+out
+gen
+### Go template
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
```

Comment:
As I told yesterday, it's a bad practice to use gitignore templates.... Since you're using GoLand, can you remove these lines only?